skbio/stats/composition.py

```diff
@@ -101,12 +101,12 @@
 #
 # The full license is in the file COPYING.txt, distributed with this software.
 # ----------------------------------------------------------------------------
-
 import numpy as np
 import pandas as pd
 import scipy.stats
 import skbio.util
 from skbio.util._decorator import experimental
+from skbio.stats.distance import DistanceMatrix, DistanceMatrixError
```

Comment:
```suggestion
from skbio.stats.distance import DistanceMatrix
```
`DistanceMatrixError` should be imported in the `test_composition.py`